extensions/v2ex-news/package.json

```
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "v2ex-news",
+  "title": "v2ex-news",
```

```suggestion
  "title": "V2EX",
```
I think you should move the -news part (and rename extension folder)

---

i changed the name , but folder name cannot change, because there has a folder named `v2ex` exist.